### PR TITLE
Fix/concert details reserve organizer

### LINKED ISSUE #26: Fix/ConcertDetails - Reserve button - organizer
- [x] Fix Reserve button: It should redirect to the reserve form page
- [x] Fix Organized by: It should display the Organizer Name
- [x] Fix Back button: It should redirect to home page

### PR DESCRIPTION
**Pull request by Diegogagan2587**
# Implemented Changes
- [x] ConcertDetailsPage.jsx
    - [x] Update `go back` or `Left button` to redirect to home `/`
    - [x] Fix Error: Showing previous rendered concert, instead will show a loading screen when still loading
    - [x] ItemDataPanel.jsx
        - [x] Update Reserve button for redirecting to ReservePage
        - [x] display organizer.name
        
 _Note_ The reservation form will display a preselected concert, but that feature will be implemented in another pull, this component only will redirect for now.

### CODE REVIEW

File: src/components/ItemDataPanel.jsx [hunk 1]
```diff
@@ -1,8 +1,16 @@
 import{ PropTypes } from 'prop-types';
 import RoundedButton from './buttons/RoundedButton';
+import {useNavigate} from 'react-router-dom';
 
 function ItemDataPanel(props) {
   const { title, organizer, description, price, date, city } = props.concert;
+  const navigate = useNavigate();
+
+  const handleReserveClick = () => {
+    console.log("Reserve button clicked")
```
Comment (danifromecuador) [inline src/components/ItemDataPanel.jsx]: - [ ] Please delete this log to the console, using console.log is good for debugging your work in the development environment but not in the production environment.

**Review by danifromecuador — CHANGES_REQUESTED** [review]
Hi @Diegogagan2587  

## STATUS: Changes required!
Great job so far 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!

almost

## Highlights
 - The back button or left button and the reserve button are working fine. Great job!   ✅
 
## Required Changes ♻️
- [ ] Please delete this log to the console in your code. Using this is good for the development environment but not for the production environment. 
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @danifromecuador .

File: src/components/ItemDataPanel.jsx (hunk 1 above)
Comment (Diegogagan2587) [reply src/components/ItemDataPanel.jsx]: Thanks for pointing that out, I'll do it right now!

Comment (Diegogagan2587) [reply src/components/ItemDataPanel.jsx]: Removed! @danifromecuador

**Review by danifromecuador — APPROVED** [review]
Hi @Diegogagan2587

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

## Highlights
 - All project requirements have been implemented ✅

chuck-norris-thumbs-up

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @danifromecuador .

**Pull request merged**